setup.cfg

```diff
@@ -1,6 +1,5 @@
 [metadata]
 name = traceback2
-version = 1.0.0
```

Comment:
We should probably fix the version number here. I'd be happy to merge a PR if you have some time submit one.